src/Http/Response.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\Http;
+
+use Psr\Http\Message\ResponseInterface;
+use Yoti\DocScan\Exception\DocScanException;
+
+class Response
+{
+    /** HTTP messages */
+    private const BAD_REQUEST_MESSAGE = 'Failed validation - ';
+    private const UNAUTHORIZED_MESSAGE = 'Failed authorization with the given key - ';
+    private const INTERNAL_SERVER_MESSAGE_ERROR = 'An unexpected error occurred on the server - ';
+    private const DEFAULT_MESSAGE = 'Unexpected error - ';
+
+    /** HTTP codes */
+    private const BAD_REQUEST_CODE = 400;
+    private const UNAUTHORIZED_CODE = 401;
+    private const INTERNAL_SERVER_CODE = 500;
+
+    /**
+     * @param ResponseInterface $response
+     * @throws DocScanException
+     */
+    public static function createYotiExceptionFromStatusCode(ResponseInterface $response): void
```

Comment:
Doesn't look like this is used anywhere but the tests. We need to include the HTTP response on any exception that we return to clients